### DISCUSSION TITLE
#9. VATZ biweekly meeting at 2022-05-25 13:00:00(KST)

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#116] [General]
# Retrospect

![image](https://user-images.githubusercontent.com/6308023/168193911-111cce89-cfb2-48ee-b69a-2ecec44a801a.png)
![image](https://user-images.githubusercontent.com/6308023/168192427-811f8bda-5375-4e06-b976-69b4ef71e32f.png)
+1

> 8 / 14 

- Closed 
   - #72 
   - #79
   - #84
   - #95
   - #100
   - #110
   - #104 
   - #106
  
- Moved to Next Sprint
   - #101 
   - #85
   - #87
   - #69
   - #98 
   - #113 

---

# Announcement

# New Updated

# Discussion

**Comment by xellos00** [comment]
# 동요

- Development
   - https://github.com/dsrvlabs/vatz/issues/98
     - https://github.com/dsrvlabs/vatz-plugin-near

- Documentation
   - #96
    
- Discussion
   - https://github.com/dsrvlabs/vatz/issues/75
   - https://github.com/dsrvlabs/vatz/issues/101
   - https://github.com/dsrvlabs/vatz/issues/113

**Comment by gnongs** [comment]
# 진홍
- Development
  - #80 
  - #127 
  - https://github.com/dsrvlabs/vatz/issues/129

**Comment by rootwarp** [comment]
# 준교

- Development
  - https://github.com/dsrvlabs/vatz/issues/122
  - https://github.com/dsrvlabs/vatz-plugin-matic/issues/2
  - https://github.com/dsrvlabs/vatz/issues/123

- Discussion
  - https://github.com/dsrvlabs/vatz/issues/108
  - https://github.com/dsrvlabs/vatz/issues/125

**Comment by kim201212** [comment]
# 형기

- Discussion
   - #87 
   - #119 
   - #120

**Comment by meetrick** [comment]
# 황제
- Development
  - #121

**Comment by heejin-github** [comment]
# 희진
- Development
  - #118 
    - https://github.com/dsrvlabs/vatz-plugin-solana/pull/2
    - https://github.com/dsrvlabs/vatz-plugin-solana/pull/3
  - #131 
    - #132

**Comment by hqueue** [comment]
# 형규

- https://github.com/dsrvlabs/vatz/issues/114
- Suggestions
   - Let's ask question and discuss code after pushing WIP code in forked repo branch.
   - Share how to review and close issues (suggestion)
       - How to review https://github.com/dsrvlabs/validators-documents/pull/1
       - Let's exploit `closes` keyword. It works between different repositories, e.g. #114 and https://github.com/dsrvlabs/validators-documents/pull/1

**Comment by xellos00** [comment]
- PR Policy 
  - First Review approved / First Merge.
  - Delete Branch that has been merged.
  - if there's anyone with comment and doesn't comment on PR must reply when it has fixed and resolve/close their comment or suggestion. 
      - Tracking PR status by person who requested PR.
  - Must include either of `closes` or `related` 
    - #{link} to close when PR is merged when this PR finishes issue.

**Reply by xellos00** [reply]
@xellos00 record this on doc on Vatz
